gdeploy.spec

```diff
@@ -70,6 +70,12 @@ mkdir -p %{buildroot}/%{python2_sitelib}/ansible/plugins/callback
 install -p -m 644 plugins/callback/gdeploy.py \
         %{buildroot}/%{python2_sitelib}/ansible/plugins/callback/
 
+# Install the vdo service file
+# To-Do Install service file in the following path
+# /usr/lib/systemd/system/vdo.service 
+
```

Comment:
Why add a TODO? Insert a cp statement.

Reply:
this is part of another commit. Will remove it from this commit.

Reply:
VDO service file part of this commit. 
https://github.com/gluster/gdeploy/pull/504